packages/flutter_tools/lib/src/commands/apk.dart

```diff
@@ -66,19 +67,23 @@ class _ApkBuilder {
     return (_androidJar.existsSync() && _aapt.existsSync() && _zipalign.existsSync());
   }
 
-  void package(File outputApk, File androidManifest, Directory assets, Directory artifacts) {
-    _run(_aapt.path, [
+  void package(File outputApk, File androidManifest, Directory assets, Directory artifacts, Directory resources) {
+    List<String> packageArgs = [_aapt.path,
       'package',
       '-M', androidManifest.path,
       '-A', assets.path,
       '-I', _androidJar.path,
       '-F', outputApk.path,
       artifacts.path
-    ]);
+    ];
+    if (resources.existsSync()) {
```

Comment:
This is slightly sketchy, and I should probably do this check outside of this function and instead of this trigger based on if resources != null.  That way we would not silently fail if a caller passes an invalid resources directory and it would be teh caller's responsibility to assure we either got a valid one or null.